### PR TITLE
add orcid id to hotjar

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/qj8NIADs/7211-collect-orcid-id-via-hotjar-identity-api

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Collect ORCID iD via Hotjar identity API](https://trello.com/c/qj8NIADs/7211-collect-orcid-id-via-hotjar-identity-api)

File: orcid-web/src/main/resources/freemarker/layout/hotjar.ftl [hunk 1]
```diff
@@ -9,6 +9,11 @@
 	        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
 	        a.appendChild(r);
 	    })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
-	</script> 
+	</script>
+	<script>
+	    if (orcidVar && orcidVar.orcidId) {
+	        window.hj('identify', orcidVar.orcidId, {});
+	    }
```
Comment (leomendoza123) [inline orcid-web/src/main/resources/freemarker/layout/hotjar.ftl:16]: @DanielPalafox Once the user logs out we need to send null as the identity according to this docs

https://help.hotjar.com/hc/en-us/articles/360033640653-Identify-API-Reference#handling_user_ids

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: orcid-web/src/main/resources/freemarker/layout/hotjar.ftl (hunk 1 above)
Comment (leomendoza123) [reply orcid-web/src/main/resources/freemarker/layout/hotjar.ftl:16]: We also need to use the "realOrcid" here

Comment (DanielPalafox) [reply orcid-web/src/main/resources/freemarker/layout/hotjar.ftl:16]: Done!

**Review by leomendoza123 — APPROVED** [review]

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**